src/power_divergence.jl

```diff
@@ -0,0 +1,349 @@
+export PowerDivergenceTest, ChisqTest, MultinomialLRT
+
+boundproportion(x) = max(min(x,1),0)
+
+immutable MultinomialData
+  observed::Vector
```

Comment:
These things should probably be parametric types. As is, they're black holes for the type inference system.